jest.config.js

```diff
@@ -1,4 +1,14 @@
 const lernaJson = require("./lerna.json");
+const e2e = require("minimist")(process.argv.slice(2))["e2e"];
+
+const isUnitTest = e2e !== "true";
+
+console.log(`Starting ${isUnitTest ? "unit" : "e2e"} tests...`);
+
+const UNIT_TEST_PATTERNS = [
+  "<rootDir>/packages/**/__tests__/**/*spec.[jt]s?(x)"
+];
+const E2E_TEST_PATTERNS = ["<rootDir>/packages/**/__e2e__/**/*spec.[jt]s?(x)"];
```

Comment:
```suggestion
const E2E_TEST_PATTERNS = ["<rootDir>/packages/**/__e2e__/**/*.e2e.spec.[jt]s?(x)"];
```

Reply:
unfortunately, this could cause a scenario when someone would add tests, fire them, think that they're passing and push it